app/views/requests/_hidden_form_fields.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= f.hidden_field :item_id %>
-<%= f.hidden_field :origin %>
-<%= f.hidden_field :origin_location %>
+<%= f.hidden_field :item_id, autocomplete: 'off'  %>
+<%= f.hidden_field :origin, autocomplete: 'off' %>
+<%= f.hidden_field :origin_location, autocomplete: 'off' %>
```

Comment:
Sorry, I was a bit overzealous with these after reading "patrons submitting forms with data they may not even be aware is present" in the [issue in JIRA](https://jirasul.stanford.edu/jira/browse/SW-1794). 

After further reading it looks like library_id is the only field causing errors. I'll dial it back to library_field in that case.